react_practice/react-apps/waitlist-page-v1/src/PrimaryBtn.css

```diff
@@ -1,21 +1,18 @@
-.primary-btn-title {
+.primary-btn {
+  border: 2px solid #1e90ff;
   padding: 10px 18px;
   background-color: #1e90ff;
   color: #fff;
   font-size: 12px;
   font-weight: 700;
-  max-width: 48%;
   border-radius: 5px;
   text-align: center;
-  margin: 10px auto 50px;
 }
 
-.primary-btn-title:hover {
-  padding: 8px 16px;
-  border: 2px solid black;
+.primary-btn:hover {
+  border-color: black;
   background-color: #fff;
   color: black;
-  font-size: 12px;
```

Comment:
As a general rule of thumb, you only want to add CSS properties to the `:hover` state that override the base styles from the base style rule. You'll see that you've already defined `font-size` above, so no need to repeat yourself here. This is where the cascade part of C(ascading)S(tyle)S(heets) comes into play! This makes it useful for you so that you do not have to repeat something that remains the same.